tox.ini

```diff
@@ -29,3 +29,14 @@ commands =
         --non-interactive \
         --ignore-missing-imports \
         --package {posargs:"atomic_reactor"}
+
+[paths]
```

Comment:
shouldn't this be `[coverage:paths]`?

Reply:
Removed. It's actually useless for this pr.